### PR TITLE
Fixes #6929: Windows is unable to load ncf directories

### PR DESCRIPTION
**Pull request by Kegeruneku**
https://www.rudder-project.org/redmine/issues/6929

Replacing previous PR: https://github.com/Normation/rudder-techniques/pull/713

### CODE REVIEW

**Comment by Kegeruneku** [comment]
Do not merge yet

**Comment by Kegeruneku** [comment]
PR updated

**Comment by Kegeruneku** [comment]
PR updated

**Comment by Kegeruneku** [comment]
PR updated

**Comment by Kegeruneku** [comment]
PR updated

File: techniques/system/common/1.0/promises.st [hunk 1]
```diff
@@ -105,31 +105,57 @@ bundle common va
...
+      "ncf_common_inputs_50" slist => splitstring(execresult("c:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command \"ls -Path 'c:\Program Files\Rudder\var\ncf\common\50_techniques' -Name *.cf -Recurse | % { Write-Host c:\Program Files\Rudder\var\ncf\common\50_techniques\$_ }\"", "useshell"), "\n", 10000);
+      "ncf_common_inputs_60" slist => splitstring(execresult("c:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command \"ls -Path 'c:\Program Files\Rudder\var\ncf\common\60_services' -Name *.cf -Recurse | % { Write-Host c:\Program Files\Rudder\var\ncf\common\60_services\$_ }\"", "useshell"), "\n", 10000);
+
+      "ncf_local_inputs_10" slist => splitstring(execresult("c:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command \"ls -Path 'c:\Program Files\Rudder\var\ncf\local\10_ncf_internals' -Name *.cf -Recurse | % { Write-Host c:\Program Files\Rudder\var\ncf\local\10_ncf_internals\$_ }\"", "useshell"), "\n", 10000);
```
Comment (ncharles) [inline techniques/system/common/1.0/promises.st]: could you add the -ErrorAction SilentlyContinue on each ls ?

**Comment by Kegeruneku** [comment]
PR updated

**Pull request merged**